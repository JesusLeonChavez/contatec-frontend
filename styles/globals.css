:root{
  --primary: #706F6F;
  --secondary: #304659;
}

html,
body {
  padding: 0;
  margin: 0;

}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
  font-family: 'Poppins', sans-serif;
}

.generalWrapper{
  max-width: 1300px;
  margin: 0 auto;
  overflow: auto;
  padding: 0 40px;
}

.flexCenterCustom{
  display: flex;
  align-items: center;
  justify-content: center;
}


/*----------------Swiper-----------------------*/
.swiper-container {
  width: 100%;
  height: 100%;
}

.swiper-slide {
  text-align: center;
  font-size: 18px;
  background: red;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}

.slide-2{
  background: green;
}

.mySwiper  .swiper-button-next, .mySwiper  .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper .swiper-button-next{
  top: 800px;
  right: 100px;
}

.mySwiper .swiper-button-prev{
  top: 800px;
  right: 180px;
  left: auto;
}

.mySwiper .swiper-button-next::after,.mySwiper .swiper-button-prev::after{
  font-size: 1rem;
}


.mySwiper2{
  width: 200px;
}
.mySwiper2 .swiper-container{
  width: 100%;
  height: 100%;
}
.mySwiper2 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: pink;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper2 .slide-2{
  background: blue;
}


.mySwiper3{
  width:100%;
}
.mySwiper3 .swiper-container{
  width: 300px;
  height: 100%;
}
.mySwiper3 .swiper-slide{
  text-align: center;
  font-size: 18px;
  background: #fff;
  background: purple;
  height: 500px;
  /* Center slide text vertically */
  display: -webkit-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  align-items: center;
}
.mySwiper3 .slide-2{
  background: blue;
}

.mySwiper3 .swiper-button-next, .mySwiper3 .swiper-button-prev {
  color: white;
  border: 1px solid white;
  border-radius: 50%;
  padding: 1.3rem;
}

.mySwiper3 .swiper-button-next{
  top: 450px;
  right: 100px;
}
.mySwiper3 .swiper-button-prev{
  top: 450px;
  right: 180px;
  left: auto;
}

.mySwiper3 .swiper-button-next::after, .mySwiper3 .swiper-button-prev::after{
  font-size: 1rem;
}




