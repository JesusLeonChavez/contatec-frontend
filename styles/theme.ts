import { extendTheme } from '@chakra-ui/react'
import { ButtonStyles as Button } from './components/buttonStyles'
export const newTheme = extendTheme({
  colors: {
    primary: '#EB0029',
    secondary: '#FF6F91',
    highlight: '#00C9A7'
  },
  components: {
    Button
  }
})
