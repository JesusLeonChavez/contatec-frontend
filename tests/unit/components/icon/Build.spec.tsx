 
 
 
/**

 * @jest-environment jsdom

 */

 import { render } from "@testing-library/react"
 import "@testing-library/jest-dom"
 import { newTheme } from "../../../../styles/theme"
 import { ChakraProvider } from "@chakra-ui/react"
 import Build from "../../../../components/Icon/Build"
 
 const ChakraRenderer = ({ children }) => {
   return <ChakraProvider theme={newTheme}>{children}</ChakraProvider>
 }
 
 describe("Build", function () {
   test("debe renderizarse el componente", () => {
     const component = render(<Build />, {
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       wrapper: ChakraRenderer
     })
 
     expect(component).toMatchSnapshot()
   })
 })
 
