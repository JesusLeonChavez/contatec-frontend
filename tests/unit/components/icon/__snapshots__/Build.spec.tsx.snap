// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Build debe renderizarse el componente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="chakra-ui-light"
  >
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <svg
        fill="none"
        height="1em"
        viewBox="0 0 27 27"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M15.2 10.797l11.014 11.014c.485.485.485 1.25 0 1.71l-2.79 2.79a1.208 1.208 0 01-1.71 0l-11.05-11.05c-2.814 1.055-6.101.461-8.357-1.795-2.79-2.778-3.045-7.132-.789-10.213l4.646 4.645 1.722-1.71-4.657-4.67c3.093-2.256 7.435-2.001 10.225.789a7.89 7.89 0 011.746 8.49zm-4.1 1.48l11.475 11.475 1.067-1.08L12.18 11.21a5.276 5.276 0 001.067-2.378 5.42 5.42 0 00-1.504-4.803 5.469 5.469 0 00-4.172-1.59l3.748 3.749-5.143 5.143-3.748-3.748a5.419 5.419 0 001.6 4.172c1.262 1.262 2.997 1.76 4.647 1.517a5.304 5.304 0 002.425-.995z"
          fill-rule="evenodd"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="1em"
      viewBox="0 0 27 27"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M15.2 10.797l11.014 11.014c.485.485.485 1.25 0 1.71l-2.79 2.79a1.208 1.208 0 01-1.71 0l-11.05-11.05c-2.814 1.055-6.101.461-8.357-1.795-2.79-2.778-3.045-7.132-.789-10.213l4.646 4.645 1.722-1.71-4.657-4.67c3.093-2.256 7.435-2.001 10.225.789a7.89 7.89 0 011.746 8.49zm-4.1 1.48l11.475 11.475 1.067-1.08L12.18 11.21a5.276 5.276 0 001.067-2.378 5.42 5.42 0 00-1.504-4.803 5.469 5.469 0 00-4.172-1.59l3.748 3.749-5.143 5.143-3.748-3.748a5.419 5.419 0 001.6 4.172c1.262 1.262 2.997 1.76 4.647 1.517a5.304 5.304 0 002.425-.995z"
        fill-rule="evenodd"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
